config/routes.rb

```diff
@@ -3,4 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  root 'landing#index'
+
+  post '/register', to: 'users#new'
```

Comment:
I think the `post` verb goes to a `create` action? 
My suggestion is to have `get '/register', to: 'users#new'` and have the post to create. Then we wouldn't need line 10?